### PR TITLE
Codeface/neutron univarsaladdr convert

### PR DESCRIPTION
**Pull request by Earlz**

### CODE REVIEW

File: src/neutron/universaladdr.h [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    inline void setVersion(Version version)         { m_version = version; }
+    void setData(const std::vector<unsigned char> &data);
+    void setData(const unsigned char *data, size_t size);
+    int setHex(const char* psz);
+    int setHex(const std::string &str);
+
+    std::string getHex();
+
+private:
+    Version m_version;
```
Comment (Earlz) [inline src/neutron/universaladdr.h:49]: Where is the Version class defined? I don't see it here

Comment (icodeface) [reply src/neutron/universaladdr.h:49]: it is inside the class UniviersalAddress, see https://github.com/qtumproject/qtum-neutron/blob/bb1f2981d685a651c27982505edcfaaf24f65f91/src/neutron/universaladdr.h#L13

Comment (Earlz) [reply src/neutron/universaladdr.h:49]: Oh, doh! I was thought this was a class with a constructor or something.  Code looks good so far, just figure out some tests

**Comment by Earlz** [comment]
Closing so @icodeface can make his own PR

**Pull request closed without merging**